content/blog/posts/the-making-of-the-imagination-o3de-ray-tracing-gdc-2022-demo.md

```diff
@@ -0,0 +1,168 @@
+---
+title: "Repost: The making of the Imagination O3DE ray tracing GDC 2022 demo"
+date: 2022-12-07 
+slug: imagination-o3de-ray-tracing 
+author: Alekos Caporali
+blog_img: "/images/blog/imagination-demo/image1.jpg"
+full_img: ""
+---
+
```

Comment:
Please include a brief statement at the front of each blog post stating who it is from, what is covered, and why somebody might care!

Reply:
I did not want to change the blog from the default one. Should we still draft something up? 

The blog was created outside of the Foundation.

Reply:
Let's not make it a new requirement right now, but I would recommend future blogs have a statement about who the author is.